kuma/wiki/forms.py

```diff
@@ -327,63 +356,57 @@ def clean_current_rev(self):
             # If there's no document yet, just bail.
             return current_rev
 
-    def save_section(self, creator, document, **kwargs):
```

Comment:
It did at a time but it was causing bugs. I'm fine to remove it all now. I have an idea for a front-end experiment that can validate demand for the feature in the future. If we get that validation, we can look at re-implementing it on this new and better code.